products/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+
+from users.models  import User
+
+class Category(models.Model):
+    name   = models.CharField(max_length=50)
+
+class Product(models.Model):
+    korean_name     = models.CharField(max_length=20)
+    english_name    = models.CharField(max_length=20)
+    thumbnail_image = models.URLField(max_length=200)
+    price           = models.DecimalField(max_digits=10, decimal_places=2)
+    created_at      = models.DateTimeField(auto_now_add = True)
+    updated_at      = models.DateTimeField(auto_now = True)
```

Comment:
`=` 앞뒤간격 전체적으로 통일해주세요!
```suggestion
    created_at      = models.DateTimeField(auto_now_add=True)
    updated_at      = models.DateTimeField(auto_now=True)
```

Reply:
수정 완료했습니다!